test/TimedTestTests.h

```diff
@@ -44,11 +44,43 @@
 /// @brief Header for unit tests for the testing framework, but only the time sensitive parts
 
 #include "MezzTest.h"
+#include "TimingTools.h"
 
 #include <stdexcept>
 #include <thread>
 #include <random>
+#include <vector>
+#include <numeric>
 
+/// @brief A class for
```

Comment:
for.......?!  The suspense is killing me.

Reply:
More 3am coding to blame. Fixed.